cfme/common/provider.py

```diff
@@ -756,7 +756,7 @@ def _fill_credential(form, cred, validate=None):
                 'token_verify_secret': cred.verify_token,
                 'validate_btn': validate
             })
-            if validate:
+            if validate_hawkular:
```

Comment:
Which providers? I work only with the Container Providers and don't really know what happens outside them